appium/webdriver/webdriver.py

```diff
@@ -940,11 +940,11 @@ def end_test_coverage(self, intent, path):
         }
         return self.execute(Command.END_TEST_COVERAGE, data)['value']
 
-    def lock(self, seconds):
+    def lock(self, seconds=0):
```

Comment:
Thanks! `zero` is also good, but could you defien `None` like below? (The behaviour is the same as `zero` or less case.)

```python
def lock(self, seconds=None):
    if seconds is None:
        self.execute(Command.LOCK)
    else:
        self.execute(Command.LOCK, {'seconds': seconds})

return self
```

If you have no time to update, it's fine for current way.